src/Search.php

```diff
@@ -6771,7 +6783,16 @@ public static function giveItem(
                         }
                         return $out;
                     } else if (($so["datatype"] ?? "") != "itemlink" && !empty($data[$ID][0]['name'])) {
```

Comment:
The `if { ... }` block should be updated too.

Reply:
fixed